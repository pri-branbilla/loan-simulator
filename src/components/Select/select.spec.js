import React from 'react'
import { Select } from '.'
import renderer from 'react-test-renderer'

const options = [
  {
    value: 1,
    label: "test 1"
  },
  {
    value: 2,
    label: "test 2"
  }
]

it('renders correctly', () => {
  const tree = renderer.create(
    <Select
      inputId="inputId"
      label="Test"
      options={options}
    />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
