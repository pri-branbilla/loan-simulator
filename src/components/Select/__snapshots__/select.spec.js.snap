// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="field"
>
  <label
    htmlFor="inputId"
  >
    Test
  </label>
  <select
    id="inputId"
    name="inputId"
    onChange={[Function]}
    required={true}
  >
    <option
      value={1}
    >
      test 1
    </option>
    <option
      value={2}
    >
      test 2
    </option>
  </select>
</div>
`;
